From 3589929a7495231887e6513af71c28231eeb66cb Mon Sep 17 00:00:00 2001
From: thanhbui <thanh.bui.fz@renesas.com>
Date: Wed, 14 Oct 2020 16:55:22 +0700
Subject: [PATCH 143/433] ARM: dts: r8a77470: Bind watchdog to SYSC PM domain

RZ/G1C cannot reboot when using CPG power-domain for watchdog (rwdt).
This patch changes the power-domain to SYSC R8A77470_PD_ALWAYS_ON to
fix this issue.

Signed-off-by: thanhbui <thanh.bui.fz@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index d903497..5814648 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -106,7 +106,7 @@
 				     "renesas,rcar-gen2-wdt";
 			reg = <0 0xe6020000 0 0x0c>;
 			clocks = <&mstp4_clks R8A77470_CLK_RWDT>;
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
-- 
2.7.4

